app/_scss/_theme_vars.scss

```diff
@@ -61,7 +61,7 @@ $Theme-font-sizes: (
         line-height: 1.86 // 26px / 14px
       ),
       large: (
-        font-size: 20px,
+        font-size: 18px,
```

Comment:
Ok. I was just pointing out that this is used in multiple places, because it was something I messed up earlier.